everett/manager.py

```diff
@@ -71,6 +97,26 @@ def get_parser(parser):
     return parser
 
 
+def get_key_from_dict(configs, key, namespace=None):
+    """Return the value of a key from the given dict respecting namespaces.
+
+    Data can also be a list of data dicts.
+    """
+    # if it barks like a dict, make it a list
+    if hasattr(configs, 'get'):
```

Comment:
We're not calling `.get()`, so it's probably better to test for the thing we're calling. I'm pretty sure for both Python 2 and Python 3 it is `__getitem__`. We should double-check that that's what gets called, though.